### PR TITLE
[Composer] Improve install instructions on install splash screen

### PR DESCRIPTION
**Pull request by andrerom**
| Question           | Answer
| ------------------ | ------------------
| **Bug/Improvement**| yes
| **Target version** | `7.5`
| **BC breaks**      | no
| **Tests pass**     | no
| **Doc needed**     | _maybe_

We had a case where a developer tried to use `server:run` for remote dev server.

This tries to avoid such misunderstandings by:
- mentioning it's local only
- expand on what it is, and its limitations so new devs can better make judgment of when to use it and not.

Also expand on what the alternative specifically is, as most have heard about Nginx/Apache.

**TODO**:
- [x] Implement feature / fix a bug.
- [x] Implement tests.
- [x] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

### CODE REVIEW

File: eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php [hunk 1]
```diff
@@ -131,16 +131,18 @@ public static function installWelcomeText(Event $event)
...
-<options=bold>Quick dev/test install:</>
+<options=bold>Quick install to test in local dev environment:</>
 <comment>    $  export SYMFONY_ENV="dev"</comment>
 ${installCommandText}
 <comment>    $  php ${consoleDir}/console server:run</comment>
 
 Note:
-- Instructions above assume the CLI user you execute these commands with is the same one that extracted/installed the software.
-- The last command will give you the url to the frontend of the installation, add "/admin" to reach backend.
+- Instructions assume CLI user you execute commands with is the same one that extracted/installed the software.
```
Comment (DominikaK) [inline eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php]:
```suggestion
- The instructions assume the CLI user you execute commands with is the same one that extracted/installed the software.
```

File: eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php [hunk 2]
```diff
@@ -131,16 +131,18 @@ public static function installWelcomeText(Event $event)
...
+- "server:run" command will:
```
Comment (DominikaK) [inline eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php]:
```suggestion
- The "server:run" command will:
```

File: eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php [hunk 3]
```diff
@@ -131,16 +131,18 @@ public static function installWelcomeText(Event $event)
...
+  - Start PHP's built in web server. A single process, local, HTTP/1.x only, mainly suitable for quick demo.
```
Comment (DominikaK) [inline eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php]:
```suggestion
  - Start PHP's built-in web server. It is a single process, local, HTTP/1.x only, mainly suitable for a quick demo.
```

File: eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php [hunk 4]
```diff
@@ -131,16 +131,18 @@ public static function installWelcomeText(Event $event)
...
+  - Give you the url to the frontend of the installation. TIP: Add "/admin" to reach backend.
```
Comment (DominikaK) [inline eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php]:
```suggestion
  - Give you the URL to the front end of the installation. TIP: Add "/admin" to reach back end.
```

File: eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php [hunk 5]
```diff
@@ -131,16 +131,18 @@ public static function installWelcomeText(Event $event)
...
 
-For full install instructions, both for production and better performing dev setup, see:
+For full install instructions with Nginx/Apache for production, remote use, or better performing dev setup:
```
Comment (DominikaK) [inline eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php]:
```suggestion
For full installation instructions with Nginx/Apache for production, remote use, or better performing dev setup:
```

File: eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php [hunk 6]
```diff
@@ -131,16 +131,18 @@ public static function installWelcomeText(Event $event)
 
 <fg=cyan>Welcome to ${installName}!</fg=cyan>
 
-<options=bold>Quick dev/test install:</>
+<options=bold>Quick install to test in local dev environment:</>
```
Comment (DominikaK) [inline eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php]:
```suggestion
<options=bold>Quick installation to test in local dev environment:</>
```

File: eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php (hunk 1 above)
Comment (andrerom) [reply eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php]: Thanks, be aware Im on purpose try to be short here as terminal has a tendency to have fixed width.

Unsure if it matters, but now you know why I try to stay within width limits

Comment (andrerom) [reply eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php]: @alongosz Do you think width matters here?

If so, maybe we can try to rephrase the 3 remaining sentences to keep them short.

File: eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php [hunk 7]
```diff
@@ -131,16 +131,18 @@ public static function installWelcomeText(Event $event)
...
-<options=bold>Quick dev/test install:</>
+<options=bold>Quick installation to test in local dev environment:</>
 <comment>    $  export SYMFONY_ENV="dev"</comment>
 ${installCommandText}
 <comment>    $  php ${consoleDir}/console server:run</comment>
 
 Note:
-- Instructions above assume the CLI user you execute these commands with is the same one that extracted/installed the software.
-- The last command will give you the url to the frontend of the installation, add "/admin" to reach backend.
+- Instructions assume CLI user you execute commands with is the same one that extracted/installed the software.
```
Comment (DominikaK) [inline eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php]: Shorter? Yes we can ;)

```suggestion
- The instructions assume you execute commands with the CLI user that extracted/installed the software.
```

File: eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php (hunk 3 above)
Comment (andrerom) [reply eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php]: Maybe rephrase as:
```suggestion
  - Use PHP's single process, local, HTTP/1 only built-in web server, mainly suitable for testing.
```

File: eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php (hunk 5 above)
Comment (andrerom) [reply eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php]: Maybe:
```suggestion
For main installation instructions with Nginx/Apache for use in production, remote, or better performing dev setup:
```

File: eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php (hunk 1 above)
Comment (alongosz) [reply eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php]: > @alongosz Do you think width matters here?
> 
> If so, maybe we can try to rephrase the 3 remaining sentences to keep them short.

I might not know something here, but I'd say - no it shouldn't matter much[1] nowadays. My imagination is that if we had an issue with a fixed width terminal, such terminal would be 80 characters tops, which was already too long before those changes in the diff.

The question is if there was an actual issue reported in the past? That would help us determining if we should be worried about it.

[1] We shouldn't make it too long due to readability anyway, but those changes for me are quite within reasonable limits.

File: eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php (hunk 5 above)
Comment (andrerom) [reply eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php]: @DominikaK is this also ok?

File: eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php (hunk 1 above)
Comment (andrerom) [reply eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php]: > The question is if there was an actual issue reported in the past? That would help us determining if we should be worried about it.

nope

File: eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php [hunk 8]
```diff
@@ -131,16 +131,18 @@ public static function installWelcomeText(Event $event)
...
 Note:
-- Instructions above assume the CLI user you execute these commands with is the same one that extracted/installed the software.
-- The last command will give you the url to the frontend of the installation, add "/admin" to reach backend.
+- The instructions assume you execute commands with the CLI user that extracted/installed the software.
+- The "server:run" command will:
+  - Use PHP's single process, local, HTTP/1 only built-in web server, mainly suitable for testing.
+  - Give you the URL to the front end of the installation. TIP: Add "/admin" to reach back end.
 
-For full install instructions, both for production and better performing dev setup, see:
+For full install instructions with Nginx/Apache for production, remote use, or better performing dev setup:
```
Comment (DominikaK) [inline eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php]: Maybe even this, to get rid of the hanging "for":
```suggestion
See main installation instructions with Nginx/Apache for production, remote use, or better performing dev setup in:
```
But both versions are fine by me.

**Comment by andrerom** [comment]
Accepted, all reviews still valid? :)
